types/cli-progress/index.d.ts

```diff
@@ -232,6 +268,7 @@ export const Format: {
     TimeFormat: typeof formatTime;
 };
 
-export class Bar extends SingleBar {}
+export class Bar extends SingleBar {
+}
```

Comment:
```suggestion
export class Bar extends SingleBar {}
```
don't change this, IMO this is outcome default DT format enforced via prettier.